.github/workflows/release.yml

```
@@ -2,42 +2,89 @@ name: Release
+          target_commitish: ${{ steps.release.outputs.commit }}
+          generate_release_notes: true
+          prerelease: ${{ steps.release.outputs.prerelease }}
+
+  deploy-staging:
+    needs: retag
+    if: contains(github.ref_name, '-')
+    uses: ./.github/workflows/deploy-staging.yml
+    with:
+      image-tag: ${{ github.ref_name }}
+      app-version: ${{ github.ref_name }}
+      git-sha: ${{ steps.release.outputs.commit }}
```

actionlint (1.7.8)

context "steps" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)